py/client/pydeephaven/experimental/plugin_client.py

```diff
@@ -115,21 +129,24 @@ class PluginResponseStream:
     to, depending on the server implementation.
     """
 
-    def __init__(self, stream_resp, session: 'pydeephaven.session.Session'):
+    def __init__(self, stream_resp: Any, session: Session):
```

Comment:
Is there no better hint than `Any`?

Reply:
The resp is just an object for a generic rpc call.